README.md

```diff
@@ -10,6 +10,49 @@ intended for testing new features, and can deploy sandboxes automatically from
 GitHub pull requests.
 
 
+What does OpenCraft Instance Manager do and how to use it
```

Comment:
Should be "What Ocim does and how to use it", or just "Using the Ocim Web Console" if we move out the "what" part as a generic section, as suggested in an earlier comment.

(We can use the acronym 'Ocim' safely now because we mentioned it above).